kit/inputs/src/UITextView/UITextView.tsx

```diff
@@ -10,15 +10,29 @@ const inputTypographyStyle = StyleSheet.flatten(Typography[textViewTypographyVar
 const textViewLineHeight = inputTypographyStyle.lineHeight ?? UILayoutConstant.smallCellHeight;
 
 /**
- * The singleline input on the ios platform has a bug -
- * if `(lineHeight - fontSize) > 5` it starts to crop
- * letters from the bottom (letters like "g", "j").
- * Style prop `textAlignVertical` works only on Android.
+ * Android TextInput does not respond to lineHeight changes,
+ * so I decided just to align input text and height with regular UILabel text and height via marginVertical.
  */
-const singleLineInputLineHeightForIOS =
-    inputTypographyStyle.fontSize && textViewLineHeight - inputTypographyStyle.fontSize > 5
-        ? inputTypographyStyle.fontSize + 5
-        : textViewLineHeight;
+function fixAndroidTextInputHeight(): TextStyle {
+    const { fontSize, lineHeight } = inputTypographyStyle;
+    if (fontSize == null || lineHeight == null) {
+        return {};
+    }
+    let textInputHeight = 0;
+    /**
+     * This calculation were carried out experimentally.
+     */
+    if (fontSize <= 20) {
+        textInputHeight = 27.636363983154297;
```

Comment:
😨

Reply:
This was measured. Logic of android native TextInput size is very not obvious.